lib/helpers/property-exists.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function checkNestedProperty (obj /*, ...Properties*/) {
+  var args = Array.prototype.slice.call(arguments, 1);
+
+  for (var i = 0; i < args.length; i++) {
+    if (!obj || !obj.hasOwnProperty(args[i])) {
+      return false;
+    }
+    obj = obj[args[i]];
+  }
+  return true;
+};
```

Comment:
Would this be more useful actually extracting the property from the object and returning it, or returning `null` if it doesn't exist. I suspect that the majority of the time you actually want to retrieve the value and use it, rather than just checking if it exists.